examples/hh/to_triton.py

```diff
@@ -24,7 +24,9 @@
 args = parser.parse_args()
 
 model_name = args.checkpoint.split("/")[-1]
-device = torch.device(args.device)
+device = os.environ.get("ACCELERATE_TORCH_DEVICE", None)
+if device is None:
+    device = torch.device(args.device)
```

Comment:
This is an auxilary script, it's not meant to be run under accelerate, so this is not needed